README.rst

```diff
@@ -14,103 +14,69 @@ Microsoft Azure SDK for Python
     :target: https://ci.appveyor.com/project/lmazuel/azure-sdk-for-python/branch/master
 
 This project provides a set of Python packages that make it easy to
-access the Microsoft Azure components such as ServiceManagement, Storage\*, and ServiceBus.
+access Management of Microsoft Azure components (Virtual Machines, ...) or Runtime (ServiceBus using HTTP, Batch, Monitor).
+Complete feature list of this repo and where to find Python packages not in this repo can be found on our 
+`Azure SDK for Python features chapter on ReadTheDocs <http://azure-sdk-for-python.readthedocs.io/en/latest/index.html#features>`__.
 
 The SDK supports Python 2.7, 3.3, 3.4, 3.5 and 3.6.
 
-\*Looking for the Azure Storage client library?  It moved to a `new GitHub repository <https://github.com/Azure/azure-storage-python>`__.
-
-See important information if you're currently using this SDK < 1.0 in `this issue <https://github.com/Azure/azure-sdk-for-python/issues/440>`__.
+See important information if you're currently using the ``azure`` package < 1.0 in `this issue <https://github.com/Azure/azure-sdk-for-python/issues/440>`__.
```

Comment:
If you are currently using the ``azure`` package < 1.0 then please read important information in `this issue <https://github.com/Azure/azure-sdk-for-python/issues/440>`__."